keyboards/qwertykeys/qk65/r2/rules.mk

```diff
@@ -0,0 +1,5 @@
+MCU_LDSCRIPT = STM32F103x8_0x08006000_bootloader
+BOOTLOADER_TYPE = stm32duino
```

Comment:
BOOTLOADER_TYPE = stm32duino can move to info.json

Reply:
If I move this the compile fails sadly.

Reply:
`bootloader` in `info.json` sets `BOOTLOADER`, not `BOOTLOADER_TYPE`, and then for `BOOTLOADER = stm32duino` it is impossible to override `DFU_ARGS` and `DFU_SUFFIX_ARGS` with the proper values that match the custom bootloader used on this PCB.

One possible option here is to specify `"bootloader": "custom"` (`bootloader_jump()` as provided by `stm32duino.c` apparently does not work on this PCB anyway).

Reply:
Sorry I don't fully follow, would you like me to add `"bootloader": "custom"` to the info.json?

Reply:
> Sorry I don't fully follow, would you like me to add `"bootloader": "custom"` to the info.json?

Ideally, yes, and provide a `bootloader_jump` and `mcu_reset` function that works correctly with the bootloader that is being used here.  

Without the correct functions+handling, that is why bootmagic and `QMK_BOOT` are not working.